.gitignore

```diff
@@ -0,0 +1,16 @@
+### Node modules ###
+/node_modules
+
+### Logs ###
+/logs
+
+### Database ###
+/data
+
+### Specific files ###
+npm-debug.log
+
+### WebStorm ###
```

Comment:
Not really :) Buy I think that I will generalize it to refer to all JetBrains products changing it to something such as: ### JetBrains IDEs ###